drivers/net/ethernet/intel/ice/ice_lib.c

```diff
@@ -1665,14 +1664,16 @@ static void ice_vsi_set_rss_flow_fld(struct ice_vsi *vsi)
 			vsi_num);
 		return;
 	}
+
 	for (i = 0; i < ARRAY_SIZE(default_rss_cfgs); i++) {
 		const struct ice_rss_hash_cfg *cfg = &default_rss_cfgs[i];
+		int status;
 
-		status = ice_add_rss_cfg(hw, vsi, cfg);
+		status = ice_add_rss_cfg(hw, vsi_handle, cfg);
 		if (status)
 			dev_dbg(dev, "ice_add_rss_cfg failed, addl_hdrs = %x, hash_flds = %llx, hdr_type = %d, symm = %d\n",
-				cfg->addl_hdrs, cfg->hash_flds,
-				cfg->hdr_type, cfg->symm);
+				cfg->addl_hdrs, cfg->hash_flds, cfg->hdr_type,
+				cfg->symm);
```

Comment:
this change does cleanup formatting. If you keep it, it should be part of a different patch since netdev typically doesn't like code format changes in the same patches as new development.